config/vufind/EDS.ini

```diff
@@ -305,3 +305,19 @@ AU = None
 ;excludeLabel[] = 'Availability'
 ; This repeatable setting can be used to filter out values using group codes:
 ;excludeGroup[] = 'URL'
+
+; This section provides settings for optional caching of search requests. This
+; affects the primary Solr backend only; for caching of other backends, see the
+; appropriate backend-specific configuration files (Search2.ini, website.ini, etc.).
+[SearchCache]
+; Supported adapters: Memcached, Filesystem. Others may also work, but have not been
+; tested.
+;adapter = Memcached
```

Comment:
I wonder if we should update the adapter settings to use the fully qualified class names, for consistency with other code.

Reply:
I'm a bit torn on this. While the fully qualified class name is more exact (and perhaps consistent as well), I'd really like to move to the other direction in the general configuration files and hide the implementation details from the user when possible. And I feel that a fully qualified class name is such detail and ties the configuration too tightly to the implementation and complicated things if the implementation needs to be changed later.

Reply:
Good point, that level of detail doesn't really belong in the configuration; if we need to use FQCN, we should translate it in the code anyway... So never mind! :-)